### PR TITLE
Adds chains version in output of tkn version command

### LINKED ISSUE #1485: Add Chains version in `tkn version`
### Feature request

Add support for displaying version of Tekton Chains if it's installed in the cluster when `tkn version` is run.
Once https://github.com/tektoncd/chains/issues/336 is fixed, this can be worked on.

### Use case

`tkn version` should give users an overview of all Tekton components installed in the cluster, so if Chains is installed in the cluster, it only makes sense to include it in.

### UI Example

```terminal
$ tkn version
Client version: 0.21.0
Chains version: v0.52.0
Pipeline version: v0.28.2
Triggers version: v0.16.1
```

### PR DESCRIPTION
**Pull request by mansi103**
Closes #1485
Signed-off-by: mansi103 <mansiagarwal103@gmail.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

### CODE REVIEW

File: pkg/version/version.go [hunk 1]
```diff
@@ -178,6 +183,26 @@ func findPipelineVersion(deployments []v1.Deployment) string {
 	return version
 }
 
+// GetChainsVersion Get chains version
+func GetChainsVersion(c *cli.Clients, ns string) (string, error) {
```
Comment (piyush-garg) [inline pkg/version/version.go:187]:
```
        var version string
	configMap, err := getConfigMap(c, chainsInfo, ns)
	if err != nil {
		return "", err
	}	
	version = configMap.Data["version"]
	return version, nil
```

File: pkg/version/version.go [hunk 2]
```diff
@@ -32,13 +33,14 @@ const (
...
 	oldDashboardControllerSelector string = "app=tekton-dashboard"
+	chainsInfo                     string = "chains-info"
 	pipelinesInfo                  string = "pipelines-info"
 	triggersInfo                   string = "triggers-info"
 	dashboardInfo                  string = "dashboard-info"
 	operatorInfo                   string = "operators-info"
 )
 
-var defaultNamespaces = []string{"tekton-pipelines", "openshift-pipelines"}
+var defaultNamespaces = []string{"tekton-pipelines", "openshift-pipelines", "tekton-chains"}
```
[the remark this answers is not in the document]
Comment (piyush-garg) [reply pkg/version/version.go:43]: seems like the namespaces are fine

**Review by pradeepitm12 — APPROVED** [review]
/lgtm

**Pull request merged**